### PR TITLE
Fixed error when inline=true

### PR DESCRIPTION
**Pull request by Braste**
Solution for https://github.com/HstarComponents/ngx-ckeditor/issues/28

### CODE REVIEW

File: src/ck-editor.component.ts [hunk 1]
```diff
@@ -100,9 +100,15 @@ export class CKEditorComponent implements OnInit, OnDestroy, OnChanges, AfterVie
...
       inline: this.inline
     });
-    this.ckIns = this.inline
-      ? CKEDITOR.inline(this.ck.nativeElement, opt)
-      : CKEDITOR.replace(this.ck.nativeElement, opt);
+    if (this.inline) {
+      this.ckIns = CKEDITOR.inline(this.ck.nativeElement, opt);
+      let nativeElement = CKEDITOR.dom.element.get(this.ck.nativeElement);
+      let textArea = nativeElement.is("textarea") ? nativeElement : null;
+      this.ckIns.container = CKEDITOR.dom.element.createFromHtml('\x3cdiv contenteditable\x3d"' + !!opt.readOnly + '" class\x3d"cke_textarea_inline"\x3e' + textArea.getValue() + "\x3c/div\x3e", CKEDITOR.document);
```
Comment (hstarorg) [inline src/ck-editor.component.ts]: If the textArea is error, the `textArea.getValue()` will throw an error.

File: src/ck-editor.component.ts [hunk 2]
```diff
@@ -100,9 +100,15 @@ export class CKEditorComponent implements OnInit, OnDestroy, OnChanges, AfterVie
       fullPage: this.fullPage,
       inline: this.inline
     });
-    this.ckIns = this.inline
-      ? CKEDITOR.inline(this.ck.nativeElement, opt)
-      : CKEDITOR.replace(this.ck.nativeElement, opt);
+    if (this.inline) {
+      this.ckIns = CKEDITOR.inline(this.ck.nativeElement, opt);
+      let nativeElement = CKEDITOR.dom.element.get(this.ck.nativeElement);
+      let textArea = nativeElement.is("textarea") ? nativeElement : null;
```
Comment (hstarorg) [inline src/ck-editor.component.ts]: Should check the `nativeElement` is null?

File: src/ck-editor.component.ts [hunk 3]
```diff
@@ -100,9 +100,15 @@ export class CKEditorComponent implements OnInit, OnDestroy, OnChanges, AfterVie
       fullPage: this.fullPage,
       inline: this.inline
     });
-    this.ckIns = this.inline
-      ? CKEDITOR.inline(this.ck.nativeElement, opt)
-      : CKEDITOR.replace(this.ck.nativeElement, opt);
+    if (this.inline) {
+      this.ckIns = CKEDITOR.inline(this.ck.nativeElement, opt);
+      let nativeElement = CKEDITOR.dom.element.get(this.ck.nativeElement);
```
Comment (hstarorg) [inline src/ck-editor.component.ts]: Define with `const` if you not update the value.

**Comment by Braste** [comment]
This is the implementation how it is in ckeditor.js, i only implemented it into the component.

**Pull request closed without merging**